src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -121,6 +173,7 @@ class TestTargetSetup:
     pex_path=f'./{test_setup.requirements_pex.output_filename}',
     pex_args=test_setup.args,
     input_files=test_setup.input_files_digest,
+    output_directories=('.coverage',),
```

Comment:
Do you know what happens when not using coverage? I think we likely want this to be conditional.

Reply:
Currently it means that `TestResult._python_sqlite_coverage_file` is an empty digest, which is probably not ideal.